### PR TITLE
Initial terminology update

### PR DESCRIPTION
**Pull request by OniLink**
Initial PR for updating and modernizing terminology. Recipes have had name changed in accordance with upstream Jenkins terminology changes.

Remaining changes to be made involve
- upstream variables which must be renamed upstream
or
- the name of the primary branch on GitHub

### CODE REVIEW

File: attributes/bumpzone.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   'secrets_databag' => 'osl_jenkins',
   'secrets_item' => 'bumpzone',
   'github_url' => 'https://github.com/osuosl/zonefiles.git',
-  'dns_master' => 'dns_master',
+  'dns_primary' => 'dns_master',
```
[the remark this answers is not in the document]
Comment (OniLink) [reply attributes/bumpzone.rb:5]: BIND refers to it as the "Primary DNS" if you scroll down on [1]. I'll go ahead and update the variable values as well to finish converting over.

[1] https://www.isc.org/bind/

**Comment by ramereth** [comment]
LGTM but hold on merging for now.

**Review by ramereth — APPROVED** [review]
I think this is ready to go with a minor bump. Ping me on slack when you're ready before you deploy this.

**Comment by OniLink** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v2.14.0.  Have a nice day!

**Pull request merged**